highfidelity/hifi
# Make AssetClient thread safe

-- Make the Asset Client's callback maps thread safe.

BugzId: 470


Test plan:
- Load ATP assets with clear cache
- Upload an ATP asset
- Rename an ATP asset
- Delete an ATP asset
- Compare home content load time between current release and this PR.
    - Make sure there is no significant increase in load time
